src/css/base.scss

```diff
@@ -2270,3 +2270,8 @@ textarea::placeholder {
     left: 50%;
     transform: translate(-50%, -50%);
 }
+
+// opacity
+.dc__o-5{
```

Comment:
please rename it to `dc__opacity-0_5` or something similar

Reply:
Done